tests/Unit/Domain/Creators/Test_CartoonSphere2D.cpp

```diff
@@ -628,6 +628,35 @@ void test_sphere_factory() {
       time_map_vec, initial_expiration_times);
 }
 
+void test_sphere_option_wedge_errors() {
+  INFO("CartoonSphere2D option wedge errors");
+  using SphereMetavars = TestHelpers::domain::BoundaryConditions::
+      MetavariablesWithoutBoundaryConditions<3,
+                                             domain::creators::CartoonSphere2D>;
+  const auto inner_surface_error = Catch::Matchers::ContainsSubstring(
+      "The radius of the inner surface must be greater than zero.");
+  CHECK_THROWS_WITH(
+      ([&]() {
+        const auto sphere =
+            TestHelpers::test_option_tag<domain::OptionTags::DomainCreator<3>,
+                                         SphereMetavars>(
+                "CartoonSphere2D:\n"
+                "  InnerRadius: 0.0\n"
+                "  OuterRadius: 3.0\n"
+                "  InitialRefinement:\n"
+                "    - [2, 2]\n"
+                "    - [2, 2]\n"
+                "  InitialGridPoints: [2,3]\n"
+                "  RadialPartitioning: [1.5]\n"
+                "  UseEquiangularMap: true\n"
+                "  Interior:\n"
+                "    FillWithSphericity: 0.0\n"
+                "  TimeDependence: None\n");
+        dynamic_cast<const creators::CartoonSphere2D&>(*sphere).create_domain();
```

Comment:
Weird way to call a virtual member function, but I guess I don't actually object.